### PR TITLE
Complement entity class

### PR DESCRIPTION
**Pull request by fishpenguin**
Signed-off-by: fishpenguin <kun.yu@zilliz.com>

### CODE REVIEW

File: pymilvus/client/abstract.py [hunk 1]
```diff
@@ -184,13 +184,19 @@ def id(self):
...
+        for k, v in self._row_data.items():
+            fields.append(k)
+        return fields
 
     def get(self, field):
         return self.value_of_field(field)
 
     def value_of_field(self, field):
-        raise NotImplementedError('TODO: support field in Hits')
+        if field in self._row_data:
```
Comment (czs007) [inline pymilvus/client/abstract.py:196]: Do the fields contain the primary field?

Comment (fishpenguin) [reply pymilvus/client/abstract.py:196]: yes

**Pull request merged**